libcontainer/nsenter/nsexec.c

```diff
@@ -131,20 +140,40 @@ int setns(int fd, int nstype)
 }
 #endif
 
+void write_log_with_info(const char *level, const char *function, int line, const char *format, ...)
```

Comment:
All the new functions should be `static`.

Reply:
Done